docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+version: '3'
+
+services:
+  nginx:
+    image: nginx:1.15.3
+    depends_on:
+      - fixmystreet
+    ports:
+      - "8000:80"
+    volumes:
+      - ./conf/nginx.conf-docker:/etc/nginx/conf.d/default.conf
+    networks:
+      default:
+        aliases:
+          - nginx.svc
+  fixmystreet:
+    image: sagepe/fixmystreet:0.5
```

Comment:
Before we merge, we should build and push an "official" version into the fixmystreet namespace.